main.py

```diff
@@ -567,12 +571,12 @@ def handle_interface_up(self, event):
 
     @listen_to('.*.switch.interface.created')
     def on_interface_created(self, event):
-        """Update the topology based on a Port Create event."""
-        self.handle_interface_created(event)
+        """Update the topology based on a Port Create event.
 
-    def handle_interface_created(self, event):
-        """Update the topology based on a Port Create event."""
-        self.handle_interface_up(event)
+        It's handled as a link_up in case a switch send a
+        created event again and it can be belong to a link.
+        """
```

Comment:
If `of_core` sends a `interface.create` for a reconnecting interface we'll have to keep handling this way, otherwise we can revert this, if we have strong guarantees that it'd come as `link_down` or `link_up`